app/api/entities/entities.js

```diff
@@ -137,6 +137,7 @@ async function updateEntity(entity, _template, unrestricted = false) {
         const toSave = { ...entity };
 
         toSave.published = d.published;
+        toSave.permissions = d.permissions;
```

Comment:
@fnocetti i think there may be a better way to ignore the permissions, unsetting them or setting them to undefined should work, without having to request the originals, same for published? or is this done this way in order to be able to set a default ?

Reply:
No, the default was a precondition for this to work. I'm not sure at this point, I'll give it a try. I remember failing to do it with `published`. But I might be wrong.
I'll let you know how it goes 👍